server.js

```diff
@@ -368,20 +368,39 @@ app.get('/questionlist', function(req, res) {
         return res.redirect('/');
     }
 
-    /* If this is the first time accessing it, fetch list from database. */
+    var request = { user : req.session.user };
+
+    users.getQuestionsList(request, function(err, questionsList){
+        if (err) {
+            return res.status(500).send('Could not fetch questions list');
+        }
+
+        req.session.adminQuestionList = null;
+        var html = questionTable({
+            questions: questionsList
+        });
+
+        return res.status(200).send(html);
+    });
+
+    /* If this is the first time accessing it, fetch list from database. 
```

Comment:
Explanation of why this was commented would be nice just so when other people are working on this file they know why.

Reply:
removing this route soon. Just keeping it now for testing